### PR TITLE
Added Useful Links

### PR DESCRIPTION
**Pull request by sandeepmishratech**
Added useful links under Java and Python.

### CODE REVIEW

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.
Please read CONTRIBUTING.md guidelines!
Don't invent titles! Use the title given by the resource.

**Comment by sandeepmishratech** [comment]
I have made the changes and followed Alphabetical order.

**Comment by eshellman** [comment]
check the linter for help on alphabetization. In this case, use the title as displayed on the web page - or learn how to properly escape the pipe character in markdown.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1065,6 +1066,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Writing Advanced Applications for the Java 2 Platform](http://www.pawlan.com/monica/books/AdvBk.pdf) - Calvin Austin and Monica Pawlan (PDF)
 
 
+
+
```
Comment (LuigiImVector) [inline books/free-programming-books-langs.md:1070]:
```suggestion
```

_"1026:1-1066:144  warning  Incorrect number of blank lines between last section and next heading  blank-lines-1-0-2  remark-lint"_: you writed two more blank lines, please read [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#formatting) for more information

**Review by sandeepmishratech — COMMENTED** [review]
I have made the changes

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (sandeepmishratech) [reply books/free-programming-books-langs.md]: I have made the changes

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**